kolibri/plugins/coach/assets/test/useQuizCreation.spec.js

```diff
@@ -138,18 +133,21 @@ describe('useQuizCreation', () => {
         // Setup a mock exercise w/ some questions; update the activeSection with their values
         const exercise = generateExercise(10);
         const questions = exercise.questions;
-        updateSection(
-          { section_id: get(activeSection).section_id, questions, resource_pool: [exercise] }
-        );
+        updateSection({
+          section_id: get(activeSection).section_id,
+          questions,
+          resource_pool: [exercise],
+        });
         expect(get(activeQuestions)).toHaveLength(questions.length);
         expect(get(activeQuestions).length).not.toEqual(0);
         expect(get(activeSection).resource_pool).toHaveLength(1);
 
         // Now let's change the question count and see if the questions array is updated
         const newQuestionCount = 5;
-        updateSection(
-          { section_id: get(activeSection).section_id, question_count: newQuestionCount }
-        );
+        updateSection({
+          section_id: get(activeSection).section_id,
```

Comment:
None blocking comment:  you could use a more descriptive name for the variable `section` it is a bit too generic. It could be more descriptive if it were named something like `currentSection `or something along that line. I believe this would keep changing too.